src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issueadjournment/IssueAdjournmentNoticeAboutToSubmitHandler.java

```diff
@@ -322,13 +327,27 @@ private void updateHearingChannelAndWantsToAttend(SscsCaseData sscsCaseData) {
     private Integer handleHearingDuration(SscsCaseData caseData) {
         AdjournCaseNextHearingDurationType durationType = caseData.getAdjournment().getNextHearingListingDurationType();
 
-        if (STANDARD.equals(durationType)) {
+        if (!NON_STANDARD.equals(durationType)) {
```

Comment:
the if-block on L360 can now become an else block, or this if-block can just go into the else-block of that if statement